packages/runtime/src/runtime/app/api.ts

```diff
@@ -20,50 +21,70 @@ import { ChannelUserRepository } from '../users'
 import { container } from './inversify/app.inversify'
 import { TYPES } from './types'
 
+const limit = <T extends (...args: any[]) => any>(func: T): T => {
+  const limiter = new Bottleneck({ minTime: 333, maxConcurrent: 1 })
+  return limiter.wrap(func) as any
```

Comment:
Needs tweaking. Right now it's 3 calls per second for testing but obviously that's too low

Reply:
Needs to be tweakable by env var (by default unlimited)

Reply:
I added the SDK_RATE_LIMIT env var to configure the options with JSON